### PR TITLE
refactor css

### PR DESCRIPTION
**Pull request by marcobiedermann**
refactor css and markup for better seo and page speed
- cleanup code
- refactoring
- scss implementation
- mobile-first support
- use variables for colors
- use shorthand
- better coding style
- grunt task for compiling and minifying
- reduce http-requests for faster loading time
- image optimization
- retina "form me" ribbon
- seo friendly markup
- wai-aria tags
- fix spacing
- combine css files

Change requests from 9 to 4 and file size from 35.6kb to 27.8kb
homebrew-network

Image diff screenshot with [wraith](https://github.com/BBC-News/wraith)
Minimal difference because of fixing a spacing bug and github ribbon not loading correction in phantom.js
homebrew

### CODE REVIEW

**Comment by marcobiedermann** [comment]
@mikemcquaid ping! :)

**Comment by mikemcquaid** [comment]
Can you rebase this on `master`?

File: _layouts/index.html [hunk 1]
```diff
@@ -164,17 +184,24 @@ <h2 id="further-doc">{{ page.pagecontent.doc.further }}</h2>
               <div class="button">
                 <p><a href="https://github.com/Homebrew/homebrew/tree/master/share/doc/homebrew#readme">https://github.com/Homebrew/homebrew/tree/master/share/doc/homebrew#readme</a></p>
               </div>
+>>>>>>> master
```
Comment (xu-cheng) [inline _layouts/index.html]: You should handle this git conflict.

File: _layouts/index.html [hunk 2]
```diff
@@ -199,5 +226,6 @@ <h2 id="further-doc">{{ page.pagecontent.doc.further }}</h2>
         }
       }
     </script>
+>>>>>>> master
```
Comment (xu-cheng) [inline _layouts/index.html]: And this one.

**Comment by xu-cheng** [comment]
The rebate is incomplete. You haven't updated to the latest master.

File: css/fonts/chunkfive/SIL Open Font License 1.1.txt [hunk 3]
```diff
@@ -1,91 +0,0 @@
...
-DISCLAIMER
-THE FONT SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
-EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO ANY WARRANTIES OF
-MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT
-OF COPYRIGHT, PATENT, TRADEMARK, OR OTHER RIGHT. IN NO EVENT SHALL THE
-COPYRIGHT HOLDER BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
-INCLUDING ANY GENERAL, SPECIAL, INDIRECT, INCIDENTAL, OR CONSEQUENTIAL
-DAMAGES, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
-FROM, OUT OF THE USE OR INABILITY TO USE THE FONT SOFTWARE OR FROM
-OTHER DEALINGS IN THE FONT SOFTWARE.
```
Comment (xu-cheng) [inline css/fonts/chunkfive/SIL Open Font License 1.1.txt]: Why this license got removed? Is it that we don't use this library anymore?

**Comment by mikemcquaid** [comment]
Passing on this but will review another version that applies cleanly to master.

**Pull request closed without merging**